authors/apps/profiles/models.py

```diff
@@ -13,10 +13,33 @@ class creating a data model to hold information about users
     bio = models.TextField(blank=True)
     full_name = models.CharField(max_length=255, blank=True)
     image = models.URLField(blank=True)
+    follows = models.ManyToManyField(
+        'self',
+        related_name='followed_by',
+        symmetrical=False
+    )
+    followers_no = models.PositiveIntegerField(default=0)
```

Comment:
We do not need to store these numbers in the database, they can always be calculated when needed.

Reply:
Thanks, I have worked on it! I have used SerializerMethod as suggested. It is Perfect Now!